kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -31,25 +31,53 @@
 import org.kitesdk.data.spi.Registration;
 
 /**
- * <p>Convenience methods for working with {@link Dataset} instances.</p>
- *
+ * Methods for working with {@link Dataset} instances.
+ * <p>
+ * <b>URIs</b><br /><br />
```

Comment:
h2 leaves too much space (in fact, this evolved from h2 to one br to two br's). <p> is better, but two br's don't affect the output and should not block publication.